packages/fxa-react/components/Portal/index.test.tsx

```diff
@@ -44,3 +48,35 @@ it('renders multiple instances with the same ID to the same DOM parent', () => {
   expect(fooPortal).toContainElement(getByTestId('p2'));
   expect(fooPortal).not.toContainElement(getByTestId('p3'));
 });
+
+it('applies a11y improvements when id is set to "modal"', () => {
+  document.body.innerHTML = `
+    <div data-testid="root"></div>
+    <div data-testid="adjacent-to-root"></div>
+  `;
+
+  const headerId = 'some-header-id';
+  const descId = 'some-desc-id';
+
+  const { getByTestId } = render(
+    <Portal id="modal" {...{ headerId }} {...{ descId }}>
```

Comment:
I think this can be `{...{ headerId, descId }}`?

Reply:
Ah it totally can, you know I'm all about that syntactic sugar 🎉

Reply:
✨